migrations/20240510181122_store_nft_img_ipfs.sql

```diff
@@ -0,0 +1,15 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query';
+-- +goose StatementEnd
+
+SET search_path = devices_api, public;
+ALTER TABLE user_devices ADD COLUMN ipfs_cid VARCHAR;
```

Comment:
This looks malformed.

Reply:
what do you mean?

Reply:
StatementEnd before the SQL starts?

Reply:
I will just fix it.

Reply:
oh shoot, sorry I was looking at the sql itself, I never really pay attention to the goose stuff, thanks for fixing that. I would have, I was just confused.